homeassistant/components/shelly/__init__.py

```diff
@@ -4,8 +4,8 @@
 import contextlib
 from typing import Any, Final
 
-import aioshelly
 from aioshelly.block_device import BlockDevice
+from aioshelly.common import ConnectionOptions
 from aioshelly.exceptions import DeviceConnectionError, InvalidAuthError
 from aioshelly.rpc_device import RpcDevice, UpdateType
```

Comment:
I think I mentioned this already, but I am partial to exporting everything at top level of the module so internal rearranging can be done without a breaking change.